tools/ArduinoCsCompiler/Frontend/Frontend.csproj

```diff
@@ -9,20 +9,38 @@
     <AssemblyName>acs</AssemblyName>
     <RootNamespace>ArduinoCsCompiler</RootNamespace>
     <Product>Arduino CS Compiler</Product>
-    <Description>A translation compiler that retargets C# applications so they can run on a microcontroller using the unnamed firmware.</Description>
+    <Description>A translation compiler that retargets C# applications so they can run on a microcontroller using a special firmware. See Readme for further information.</Description>
     <Authors>The .NET Foundation community</Authors>
     <NeutralLanguage>en</NeutralLanguage>
+    <IsPackable>True</IsPackable>
+    <PackAsTool>True</PackAsTool>
```

Comment:
We should also make sure that this tool stays in preview and doesn't go stable (since it is experimental). You can accomplish that by adding the property: `<SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>` in the .csproj. This way, even when the rest of the repo builds stable packages, this tool will remain as prerelease.

Reply:
I did that, but it doesn't seem to do anything. Can you verify with a release build?